staging/src/k8s.io/apiserver/pkg/registry/rest/table.go

```diff
@@ -67,7 +67,7 @@ func (c defaultTableConvertor) ConvertToTable(ctx context.Context, object runtim
 		table.ResourceVersion = m.GetResourceVersion()
 		table.SelfLink = m.GetSelfLink()
 		table.Continue = m.GetContinue()
-		table.RemainingItemCount = m.GetRemainingItemCount()
+		table.SetRemainingItemCount(m.GetRemainingItemCount())
```

Comment:
this is why we need to make sure nil can be propagated from GetRemainingItemCount to SetRemainingItemCount without forcing a 0 to be serialized